cnd/src/network/comit_ln.rs

```diff
@@ -628,14 +636,48 @@ impl NetworkBehaviourEventProcess<oneshot_behaviour::OutEvent<finalize::Message>
     }
 }
 
+#[async_trait]
+impl InitCommunication for ComitLN {
+    async fn init_communication<T>(&self, swap_id: LocalSwapId, params: T) -> anyhow::Result<()> {
+        // TODO: Implement generic communication methods based on
```

Comment:
I tried that and got compiler problems because of `Send`/`'sync`.  Caused apparently by the MDNS, this current attempt was the second failed attempt, any more suggestions appreciated.